fluent-package/convert-artifacts-layout.sh

```diff
@@ -20,28 +24,38 @@ case $1 in
 		bullseye|bookworm)
 		    # e.g. mapping debian/pool/buster/main/t/td-agent/ => 5/debian/buster/pool/contrib/t/td-agent
 		    #      mapping debian/pool/buster/main/f/fluent-package/ => 5/debian/buster/pool/contrib/f/fluent-package
-		    mkdir -p $ARTIFACTS_DIR/5/debian/$d/pool/contrib/t/td-agent
```

Comment:
Did we forget to remove this in 6d7dfee9954375b567e691a22d05d58427824234?

Reply:
Yes,I've forgotten.